index.js

```diff
@@ -887,6 +887,19 @@ var QUERIES = [
       return result
     }
   },
+  {
+    regexp: /^supports\s+(\w+)$/,
+    select: function (context, feature) {
+      env.loadFeature(browserslist.usage, feature)
+      var usage = browserslist.usage[feature]
```

Comment:
let’s rename `browserslist.usage` too to `browserslist.cache` and clean it in `browserslist.clearCaches`

Reply:
Done.